Terraform/Home_Task_2/modules/asg/outputs.tf

```diff
@@ -1,6 +1,3 @@
-output "alb_dns_name" {
-  value = aws_lb.alb.dns_name
-}
 output "asg_id" {

```

Comment:
we don't need this if you move resource aws_autoscaling_attachmentthis block to asg module